deploy/kubernetes-1.13/pmem-csi-direct.yaml

```diff
@@ -150,9 +150,11 @@ spec:
       - name: pmem-driver
         image: 192.168.8.1:5000/pmem-csi-driver:canary
         imagePullPolicy: Always
+        terminationMessagePath: /tmp/termination-log
```

Comment:
We need to pass /dev to container, and if we do this, we (sometimes?) hit container creation error because termination-log is located on /dev/ by default. Adding this clause works around failure.

Reply:
Can you add a comment explaining that?

As it stands, `/tmp/termination-log` is useless because nothing knows that it needs to write at that location, so someone seeing this will wonder about it.